one_fm/templates/emails/leave_proposal_status.html

```diff
@@ -94,9 +94,9 @@
                 <div class="arabic">
                     <div class="section-title">مدة الإجازة الأصلية:</div>
                     <div class="details">
-                        <p><strong>اسم الموظف:</strong> {{employee_name_arabic}}</p>
+                        <p><strong>اسم الموظف:</strong> {{ employee_name_arabic if employee_name_arabic else "" }}</p>
                         <p><strong>رقم الموظف:</strong> {{employee_id}}</p>
-                        <p><strong>نوع الإجازة:</strong> {{leave_type_arabic}}</p>
+                        <p><strong>نوع الإجازة:</strong> {{_(leave_type_eng, "ar")}}</p>
```

Comment:
@samdanikouser 
1. There is no field called leave_type_eng. Just use leave_type here.
2. This title "Original Leave Duration" should come above Start date as per the format.

Reply:
3. Don't hardcode status. Status needs to come from workflow_state field.